dashboard-for-desktop/articles/dashboard-designer/appearance-customization/conditional-formatting.md

```diff
@@ -116,4 +85,19 @@ This invokes the **Edit Rules** dialog containing existing format rules for this
 |Create a new rule|Click the **Add** button and select the format condition. The **calculated by** combo box allows you to select the measure/dimension that is used to calculate the format rule. For Card, when you create a rule for delta, specify the delta type whose values are used to calculate a condition in the **using** combo box. |
 |Filter format rules by the specified data item| Use the **Filter by** combo box. |
 
-To clear all rules for the specified data item, use the **Clear Rules** button in the data item's context menu.
+The **Add** button invokes a pop-up menu where you can specify appearance settings and set the condition's value for the format rule. Available settings depend on the selected format rule.
+
+![rule_manager](../../../images/win-conditional-formatting-card-rule-manager.png)
+
+The image below displays the **Greater Than** dialog, which corresponds to the [Value](conditional-formatting/value.md) format condition for a [Grid](../designing-dashboard-items/grid.md) dashboard item. The format condition applies to the _ExtendedPrice_ data item in the **Apply to** drop-down list.
+
+ ![GreaterThanDialog](../../../images/img118555.png)
+## Dashboard Items Specific
```

Comment:
Can we make this section title more specific? For example, do you mean:

`## Dashboard Item Format Rule Specifics`